src/utils/ts.ts

```diff
@@ -6,3 +6,13 @@ src/orders/components/OrderGrantRefundPage/reducer.ts
 export function exhaustiveCheck(x: never) {
   return x;
 }
+
+export function nonNullable<T>(value: T): value is NonNullable<T> {
+  return value !== null && value !== undefined;
+}
+
+type Truthy<T> = T extends false | "" | 0 | null | undefined ? never : T; // from lodash
+
+export function truthy<T>(value: T): value is Truthy<T> {
+  return !!value;
+}
```

Comment:
**question**: are you planning to use it in the future? I don't see it used anywhere

Reply:
Those types are very usefull when you do filter, because doing `filter(Boolean)` doesn't return the proper type